handlers.go

```diff
@@ -93,6 +90,22 @@ func loginSubmit(w http.ResponseWriter, r *http.Request) {
 		log.Println("Login failed -", err)
 		http.Redirect(w, r, "/login-fail", http.StatusSeeOther)
```

Comment:
Please simply return after line 21 and remove the unnecessary else case for readability.